model/dyn_core.F90

```diff
@@ -1248,7 +1248,8 @@ subroutine dyn_core(npx, npy, npz, ng, sphum, nq, bdt, n_map, n_split, zvir, cp,
 
                                                      call timing_on('COMM_TOTAL')
 #ifndef ROT3
-    if ( it/=n_split)   &
+!    if ( it/=n_split)   &
```

Comment:
Is there a reason you are leaving this commented?  I would recommend deleting this line of code instead of commenting it out.

Reply:
Could you delete this commented code?

Reply:
Weird, I thought I had done that and the next one already... will do.